src/index.html

```diff
@@ -2,11 +2,395 @@
+      crossorigin="attr-req-value"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <label>
+        <img
+          class="logoMoyo"
+          name="logoMoyo"
+          src="images/logo.png"
+          alt="moyo"
+        >
+      </label>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#cellphones">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a
+                class="nav__link"
+                href="#laptops"
+                data-qa="nav-hover"
+              >
+                Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#photos">Фото</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="catalog">
+        <div class="card catalog__card" data-qa="card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="monitor"
+          >
+          <p class="card__apple-name">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </p>
+          <p class="card__product-code">
+            Код товара: 195434
+          </p>
+          <div class="card__rating">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="stars-text">Отзывов: 5</div>
+          </div>
+          <div class="price">
+            <p class="price__text">
+              Цена:
+            </p>
+            <p class="price__apple">
+              69 999 грн
+            </p>
+          </div>
+          <div class="card__button">
+            <a
+              class="button"
+              href="/"
+              data-qa="card-hover"
+            >
+              Купить
+            </a>
+          </div>
+        </div>
+        <div class="card catalog__card">
```

Comment:
For readability, add blank lines to separate large or logical code blocks, add them in all places in your code.